app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -532,18 +529,27 @@ public boolean onOptionsItemSelected(MenuItem inputItem)
                 }
             }
 
+            AlertDialog.Builder builder = new AlertDialog.Builder(MainActivity.this);
+            builder.setTitle(R.string.sort_by);
+
+            final View customLayout = getLayoutInflater().inflate(R.layout.sorting_option, null);
+            builder.setView(customLayout);
+            CheckBox ch = (CheckBox) customLayout.findViewById(R.id.checkBox_reverse);
```

Comment:
Please set the checkbox checked if the current direction is reversed, so that if people open the dialog after they sorted reversed it shows it checked and they can uncheck if they no longer want it reversed. Makes it clearer what the current status is.